nservicebus/azure-storage-persistence/configuration.md

```diff
@@ -1,42 +1,26 @@
 ---
 title: Azure Storage Persistence Configuration
 summary: Configuring Azure Storage as persistence
+reviewed: 2016-05-12
 tags:
 - Azure
 - Cloud
 - Persistence
 - Configuration
 ---
 
-The behavior of each persister can be controlled by specifying one of the respective configuration sections in the `app.config` and changing available properties, or using code API. The exact same settings are exposed via `app.config` file and code configuration API.
+In NServiceBus.Azure the behavior of the `AzureStoragePersister` can be controlled by working with the appropriate configuration section(s) in the `app.config` or by using the code API. Both approaches to configuring the persister can access the same configuration options.
```

Comment:
The first time I looked at it I was like "Version 1, WTF?". Why don't we start this whole document by explaining that the name of the package was changed, and so we started numbering from scratch?